transmission/userSetup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# More/less taken from https://github.com/linuxserver/docker-baseimage-alpine/blob/3eb7146a55b7bff547905e0d3f71a26036448ae6/root/etc/cont-init.d/10-adduser
+
+RUN_AS=root
+
+if [ -n "$PUID" ] && [ ! "$(id -u root)" -eq "$PUID" ]; then
+    RUN_AS=abc
+    if [ ! "$(id -u ${RUN_AS})" -eq "$PUID" ]; then usermod -o -u "$PUID" ${RUN_AS} ; fi
+    if [ ! "$(id -g ${RUN_AS})" -eq "$PGID" ]; then groupmod -o -g "$PGID" ${RUN_AS} ; fi
+
+    echo "Setting owner for transmission paths to ${PUID}:${PGID}"
+    chown -R ${RUN_AS}:${RUN_AS} ${TRANSMISSION_HOME}
+    chown ${RUN_AS}:${RUN_AS} \
```

Comment:
Should this also be -R maybe? Or is it better just leave files in there alone and just make the folder writable by the transmission user?